src/components/Skeletons/TableListItemSkeleton.tsx

```diff
@@ -1,17 +1,103 @@
 import React from 'react';
-import {Rect} from 'react-native-svg';
+import {Circle, Rect} from 'react-native-svg';
+import useThemeStyles from '@hooks/useThemeStyles';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import CONST from '@src/CONST';
 import ItemListSkeletonView from './ItemListSkeletonView';
 
 type TableListItemSkeletonProps = {
     shouldAnimate?: boolean;
     fixedNumItems?: number;
 };
 
+const circleRadius = 8;
+const padding = 12;
 const barHeight = '10';
 const shortBarWidth = '40';
 const longBarWidth = '120';
 
 function TableListItemSkeleton({shouldAnimate = true, fixedNumItems}: TableListItemSkeletonProps) {
+    const styles = useThemeStyles();
+    const {windowWidth, isSmallScreenWidth} = useWindowDimensions();
+    if (isSmallScreenWidth) {
+        return (
+            <ItemListSkeletonView
+                itemViewHeight={CONST.SEARCH_SKELETON_VIEW_ITEM_HEIGHT}
+                itemViewStyle={[styles.highlightBG, styles.mv2, styles.br3, styles.mr3, styles.ml3]}
+                shouldAnimate={shouldAnimate}
+                fixedNumItems={fixedNumItems}
+                renderSkeletonItem={() => (
+                    <>
+                        <Circle
+                            cx={padding + circleRadius}
```

Comment:
Instead of doing any fancy math here, can we just say that this should be 16x16?

Reply:
The fancy math will help me when debugging. I will remove these once we do not have a design change any more. Also, in this case, ```padding + circleRadius = 16```